apps/meteor/definition/IRoomTypeConfig.ts

```diff
@@ -105,6 +105,7 @@ export interface IRoomTypeServerDirectives {
 		sender: AtLeast<IUser, '_id' | 'name' | 'username'>,
 		notificationMessage: string,
 		userId: string,
+		langauge?: string,
```

Comment:
```suggestion
		language?: string,
```

Reply:
does the lan-gauge measure the lan? :trollface: